spec/controllers/users_controller_spec.rb

```diff
@@ -3,11 +3,63 @@
 RSpec.describe UsersController, type: :controller do
   let(:user) { create(:unconfirmed_user) }
 
-  describe "email confirmation" do
-    it "sends a confirmation email when user registers" do
-      expect {
-        UserMailer.registration_confirmation(user).deliver_now
-      }.to change(ActionMailer::Base.deliveries, :size).by(1)
+  describe "POST #create" do
+    let(:params) { { user: interactor_input.fetch(:user_params) } }
+
+    let(:interactor_input) do
+      { user_params: {
+        name:                  user.name,
+        email:                 user.email,
+        password:              user.password,
+        password_confirmation: user.password_confirmation
+        }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.